### PR TITLE
Add Delius flavour of RO service, behind feature switch

### PR DESCRIPTION
**Pull request by atodd-noms**

### CODE REVIEW

File: server/data/deliusClient.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+const { apiUrl } = config.delius
+
+module.exports = signInService => {
+  return {
+    getROPrisoners(deliusStaffCode) {
+      const path = `${apiUrl}/staff/staffCode/${deliusStaffCode}/managedOffenders`
+      const query = { current: true }
+      return deliusGet({ path, query })
```
Comment (timharrison-moj) [inline server/data/deliusClient.js]: The query parameter ?current=true is not required as this is added within the community proxy. Alternatively, I could rework the query parameters in the proxy?

File: server/data/deliusClient.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    },
+
+    getResponsibleOfficer(offenderNo) {
+      const path = `${apiUrl}/offenders/nomsNumber/${offenderNo}/responsibleOfficers`
+      const query = { current: true }
```
Comment (timharrison-moj) [inline server/data/deliusClient.js]: Same as above - ?current=true not required.

File: test/data/deliusClientTest.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  describe('getROPrisoners', () => {
+    it('should return data from api', () => {
+      fakeDelius.get(`/staff/staffCode/1/managedOffenders?current=true`).reply(200, { key: 'value' })
+
+      return expect(deliusClient.getROPrisoners('1')).to.eventually.eql({ key: 'value' })
+    })
+
+    it('should reject if api fails', () => {
+      fakeDelius.get(`/staff/staffCode/1/managedOffenders?current=true`).reply(500)
+
```
Comment (timharrison-moj) [inline test/data/deliusClientTest.js]: Same thing - ?curren=true added by the proxy. Lets discuss Monday to see where the best place for this is?

File: server/data/deliusClient.js (hunk 1 above)
Comment (atodd-noms) [reply server/data/deliusClient.js]: Done

**Pull request merged**